app/Http/routes.php

```diff
@@ -22,5 +22,27 @@
 	   'password' => 'Auth\PasswordController',
 	]);
 
+	// Starting Route of Admin and User Start
+	Route::get('/', 'HomeController@index');
 	Route::get('/home', 'HomeController@index');
+	Route::get('/admin', 'HomeController@admin');
+	// Starting Route of Admin and User End
+
+	// User Side Routes
+	Route::get('exam','user\ExamController@exam');
+	Route::get('editprofile','user\ProfileController@editprofile_user');
+	Route::get('viewresult','user\ResultController@viewresult');
+
```

Comment:
Create rout group. All admin side url should be {{base_url}}/admin/{{controller}}/{{action}}